### DISCUSSION TITLE
Met Installation.

### DISCUSSION THREAD

**Discussion by raj879423** [dtcenter/METplus#2012] [Installation] [answered]
Hello Team,

I am compiling Met-11.0.0 and i am installation through script file but it is giving me some error related to gcc version.  
 I check in google the error msg it is showing related to gcc. 

This is the error i am getting after installation of all dependencies packages for Met.
### " lto1: fatal error: bytecode stream generated with LTO version 6.0 instead of the expected 5.1"
 
Kindly let me know which GCC version and python version i have to use for compiling Met-11.0.0 latest version.

Below are link i am following for compiling met and my installation script file.

"https://metplus-training.readthedocs.io/en/latest/modules/Environment/met_installation.html"

"https://raw.githubusercontent.com/dtcenter/MET/main_v11.0/internal/scripts/installation/compile_MET_all.sh "

$ cat  install_met_env.utility04

#module use -a /cm/local/modulefiles

module use -a /home/apps/SiteSoftwares/gnu/ANACONDA3

module load gcc/6.3.0

module load gnu/anaconda3/2019.10

#export CC=gcc

#export FC=gfortran

#export CXX=c++

#export TEST_BASE=/home/raj/software/met/met_install_11.0
export TEST_BASE=/home/raj/software/met/11.0.0

#export COMPILER=gcc_6.3.0

export COMPILER=${COMPILER}gnu_6.3.0

export MET_SUBDIR=${TEST_BASE}

export MET_TARBALL=v11.0.0.tar.gz

export USE_MODULES=TRUE

export PYTHON_MODULE=gnu_anaconda3_2019.10

export MET_PYTHON=/home/apps/SiteSoftwares/gnu/ANACONDA3/2019.10

export MET_PYTHON_CC=-I${MET_PYTHON}/include/python3.7m

export MET_PYTHON_LD=-L${MET_PYTHON}/lib/python3.7/config-3.7m-x86_64-linux-gnu\ -L${MET_PYTHON}/lib\ -lpython3.7m\ -lcrypt

export SET_D64BIT=FALSE

##### I have request please provide me solution for compiling Met.
I am ready to compile met-11.0.0  with GCC along with INTEL but please let me know which icc or gcc version will compatible with MET-11.0.0.

Thanks& Regards 
Raj Sharma

**Comment by jprestop** [comment]
Hi @raj879423.  

The MET software can be compiled with many different versions of the GNU and Intel compiler and with various Python versions.  

With regard to this error:
```
" lto1: fatal error: bytecode stream generated with LTO version 6.0 instead of the expected 5.1"
```
Other users have encountered this problem before.  In those cases, the anaconda python gcc was a different version compared with what they were using to compile.  To get a successful compilation, the users switched to using the system's installed default version of Python instead to get a successful installation.  Is that an option for you?  If not, are there gcc, g++, and gfortran executables within your anaconda area that you could point to instead to try building with those?

**Comment by raj879423** [comment]
Hello @jprestop,

Greetings of the day.

First of all thank you so much for your quick response. 

As per your suggestion I have successfully compiled MET-11.0.0 with system build gcc and with python-3.6.4.

Now i am trying to compile METplus-5.0.0 but i am having small confusion i really need your help and suggestion on this.

I watched your video for reference installation of METplus latest version "https://www.youtube.com/watch?v=ap9-Fdlb7Fo&t=78s" .

So far what i understood I follow same step as per the video -: 1st I have download "METplus-5.0.0.tar.gz and sample_data-met_tool_wrapper-5.0.tgz" after that i made some changes in "METplus-5.0.0/parm/metplus_config/defaults.conf" file I put MET installed location in "MET_INSTALL_DIR =/home/raj/software/met/met_python/tar_files " then "INPUT_BASE = /home/raj/software/met/met_plus/METplus-3.1_sample_data/"

After watched your video fully. Now my confusion is you have mentioned one file under "METplus directory" that is  "hera.jpresto.video.conf" & another directory i guess is "output".

As per i understand may i know from where i will get the video so that i can create "output" directory for further compilation.

For your reference you have mention in your file "hera.jpresto.video.conf" :-

OUTPUT_BASE="scratch1/BMC/dtc/---METplus/output/video". So from where i will get this video ?

One more question, you have chosen "5.1 MET tools" for METplus-3.1 for running your script at the end.

Kindly suggest me which "MET tools" i have to select for METplus-5.0.0.

I hope you understand my query, Thank you on Advance

I am very new for this installation of MET & METplus, some how i managed for MET and successfully done now i am stuck with METplus.

Below are my downloaded software for your reference.

[raj@utility04 met_plus]$ ls
METplus-3.1_sample_data  METplus-5.0.0  sample_data-met_tool_wrapper-5.0.tgz  v5.0.0.tar.gz
[raj@utility04 met_plus]$ cd METplus-5.0.0/
[raj@utility04 METplus-5.0.0]$ ls
build_components  docs  environment.yml  internal  manage_externals  metplus  parm  produtil  pyproject.toml  README.md  requirements.txt  setup.py  ush
[raj@utility04 METplus-5.0.0]$


----
Thanks & Regards
Raj Sharma

**Reply by jprestop** [reply]
Hi @raj879423.

My apologies for any confusion in installing the METplus software.  MET_INSTALL_DIR is the MET installation directory - the location where the MET tools are installed on the system. This directory should contain at least two directories: "share" and "bin". The bin directory will contain the MET executables, such as grid_stat.  You could run the following command to find it, if you installed somewhere in "/home/raj":
```
find /home/raj -name grid_stat
```
The directory that holds "grid_stat" is the directory path you'll need to set MET_INSTALL_DIR to.  Here is a link to our documentation on [MET_INSTALL_DIR](https://metplus.readthedocs.io/en/develop/Users_Guide/systemconfiguration.html#met-install-dir).

[OUTPUT_BASE](https://metplus.readthedocs.io/en/develop/Users_Guide/systemconfiguration.html#output-base) is the directory, of your choosing, where logs and output files are written. This should be set to a path where the user running the METplus wrappers has permission to write files. The directory will be created automatically if it does not exist already.

Regarding:

> One more question, you have chosen "5.1 MET tools" for METplus-3.1 for running your script at the end.
Kindly suggest me which "MET tools" i have to select for METplus-5.0.0.

For METplus version 5.0.0, we suggest running MET version 11.0.0.   We do coordinated releases with our software.  You can view the recommended versions for our coordinated releases on our [Dowloads page](https://dtcenter.org/community-code/metplus/download).  I have attached a screenshot of the section I am referring to.

<img width="1199" alt="Screen Shot 2023-01-27 at 12 13 16 PM" src="https://user-images.githubusercontent.com/11999822/215175611-e859445c-2857-427f-9396-0e10b01d4288.png">

I hope I have answered all of your questions.  Please follow up with any other questions you have.

**Comment by raj879423** [comment]
Hello Dtcenter/Metplus,

I am very grateful and Thank you for your support.
I follow the same as per your suggestion but unfortunately I am getting
some errors while running the code.
As per i understand the error shows it is related to FCST_INPUT. Kindly
suggest or guide me to fix this problem. I am unable to find it on google
also.

Below are screenshots for your reference.
### Module used -:

$ module switch PrgEnv-cray PrgEnv-gnu
$ module load gcc/7.2.0
$ module load python-3.9.7

$ ***@***.***:~/software/met/met_plus> ls
METplus-3.1_sample_data  METplus-5.0.0  elogin.raj.video.conf  output
 sample_data-met_tool_wrapper-5.0.tgz  v5.0.0.tar.gz

$ /home/raj/software/met/met_plus/METplus-5.0.0/ush/run_metplus.py -c
/home/raj/software/met/met_plus/METplus-5.0.0/parm/use_cases/met_tool_wrapper/GridStat/GridStat.conf
-c /home/raj/software/met/met_plus/elogin.raj.video.conf



On Sat, Jan 28, 2023 at 12:45 AM jprestop ***@***.***> wrote:

> Hi @raj879423 <https://github.com/raj879423>.
>
> My apologies for any confusion in installing the METplus software.
> MET_INSTALL_DIR is the MET installation directory - the location where the
> MET tools are installed on the system. This directory should contain at
> least two directories: "share" and "bin". The bin directory will contain
> the MET executables, such as grid_stat. You could run the following command
> to find it, if you installed somewhere in "/home/raj":
>
> find /home/raj -name grid_stat
>
> The directory that holds "grid_stat" is the directory path you'll need to
> set MET_INSTALL_DIR to. Here is a link to our documentation on
> MET_INSTALL_DIR
> <https://metplus.readthedocs.io/en/develop/Users_Guide/systemconfiguration.html#met-install-dir>
> .
>
> OUTPUT_BASE
> <https://metplus.readthedocs.io/en/develop/Users_Guide/systemconfiguration.html#output-base>
> is the directory, of your choosing, where logs and output files are
> written. This should be set to a path where the user running the METplus
> wrappers has permission to write files. The directory will be created
> automatically if it does not exist already.
>
> Regarding:
>
> One more question, you have chosen "5.1 MET tools" for METplus-3.1 for
> running your script at the end.
> Kindly suggest me which "MET tools" i have to select for METplus-5.0.0.
>
> For METplus version 5.0.0, we suggest running MET version 11.0.0. We do
> coordinated releases with our software. You can view the recommended
> versions for our coordinated releases on our Dowloads page
> <https://dtcenter.org/community-code/metplus/download>. I have attached a
> screenshot of the section I am referring to.
>
> [image: Screen Shot 2023-01-27 at 12 13 16 PM]
> <https://user-images.githubusercontent.com/11999822/215175611-e859445c-2857-427f-9396-0e10b01d4288.png>
>
> I hope I have answered all of your questions. Please follow up with any
> other questions you have.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/dtcenter/METplus/discussions/2012#discussioncomment-4800029>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQGYYVGW3NIDODYLHEVW6YDWUQNGNANCNFSM6AAAAAAUAIHZQM>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Thanks & Regards,
Raj Kumar Sharma
+91- 8794236178

**Reply by georgemccabe** [reply]
Hi @raj879423, could you attach the METplus log file that includes the error you are seeing?

**Comment by raj879423** [comment]
Hello Dtcenter/Metplus,

Sorry for the late reply, we actually have a time Zone issue compared to
India

First of all I am very sorry for bothering you all for this code. .

I am  a HPC Engineer. I am compiling this application for one of our
scientists, NCMRWF. If something is required to take* "input_file"* from
the user then I am ready to ask from our scientist .

I tried my best to compile the MET-11.0.0 latest version  but I am having
problems with met-plus-5.0.0.

Please find the attached screenshot log for your reference.

Thanks once again.



On Mon, Jan 30, 2023 at 8:08 PM George McCabe ***@***.***>
wrote:

> Hi @raj879423 <https://github.com/raj879423>, could you attach the
> METplus log file that includes the error you are seeing?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/dtcenter/METplus/discussions/2012#discussioncomment-4819613>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQGYYVBFD6BW4SOZNX3RGSDWU7G4RANCNFSM6AAAAAAUAIHZQM>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Thanks & Regards,
Raj Kumar Sharma
+91- 8794236178

**Reply by jprestop** [reply]
Hi @raj879423.  You are no bother.  We are happy to help.

Unfortunately, there was no attached screenshot.  However, instead of sending a screenshot could you please attach the actual METplus log file that includes the error you are seeing along with /home/raj/software/met/met_plus/elogin.raj.video.conf?  You should be able to attach files by dragging and dropping, selecting or pasting them.

**Comment by raj879423** [comment]
Hello Dtcenter/Metplus,

Below is the METplus log for your reference.
So far I have this log only. I hope this time I have provided you with the
correct log. It might help you to understand my query.


Thank you

On Tue, Jan 31, 2023 at 9:23 PM jprestop ***@***.***> wrote:

> Hi @raj879423 <https://github.com/raj879423>. You are no bother. We are
> happy to help.
>
> Unfortunately, there was no attached screenshot. However, instead of
> sending a screenshot could you please attach the actual METplus log file
> that includes the error you are seeing along with
> /home/raj/software/met/met_plus/elogin.raj.video.conf? You should be able
> to attach files by dragging and dropping, selecting or pasting them.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/dtcenter/METplus/discussions/2012#discussioncomment-4831536>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQGYYVA2YKR276DCSG67LITWVEYQ5ANCNFSM6AAAAAAUAIHZQM>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Thanks & Regards,
Raj Kumar Sharma
+91- 8794236178

# This is the comment

[dir]
OUTPUT_BASE = /home/raj/software/met/met_plus/output/video

01/30 06:53:25.674Z metplus INFO: Running METplus v5.0.0 as user raj(743) with command: /home/raj/software/met/met_plus/METplus-5.0.0/ush/run_metplus.py -c /home/raj/software/met/met_plus/METplus-5.0.0/parm/use_cases/met_tool_wrapper/GridStat/GridStat.conf -c /home/raj/software/met/met_plus/elogin.raj.video.conf
01/30 06:53:25.675Z metplus INFO: Log file: /home/raj/software/met/met_plus/output/video/logs/metplus.log.20230130065325
01/30 06:53:25.675Z metplus INFO: METplus Base: /home/raj/software/met/met_plus/METplus-5.0.0
01/30 06:53:25.675Z metplus INFO: Final Conf: /home/raj/software/met/met_plus/output/video/metplus_final.conf.20230130065325
01/30 06:53:25.675Z metplus INFO: Config Input: /home/raj/software/met/met_plus/METplus-5.0.0/parm/metplus_config/defaults.conf
01/30 06:53:25.675Z metplus INFO: Config Input: /home/raj/software/met/met_plus/METplus-5.0.0/parm/use_cases/met_tool_wrapper/GridStat/GridStat.conf
01/30 06:53:25.675Z metplus INFO: Config Input: /home/raj/software/met/met_plus/elogin.raj.video.conf
01/30 06:53:25.714Z metplus INFO: ****************************************
01/30 06:53:25.714Z metplus INFO: * Running METplus GridStatWrapper
01/30 06:53:25.714Z metplus INFO: *  at init time: 2005-08-07 00:00
01/30 06:53:25.714Z metplus INFO: ****************************************
01/30 06:53:25.714Z metplus INFO: Processing forecast lead 12 hours
01/30 06:53:25.809Z metplus (command_builder.py:257) ERROR: (command_builder.py:656) Could not find FCST_INPUT file /home/raj/software/met/met_plus/METplus-3.1_sample_data/met_test/data/sample_fcst/2005080700/wrfprs_ruc13_12.tm00_G212 using template {init?fmt=%Y%m%d%H}/wrfprs_ruc13_{lead?fmt=%HH}.tm00_G212
01/30 06:53:25.809Z metplus (config_util.py:166) ERROR: No commands were run. Skip writing all_commands file
01/30 06:53:25.809Z metplus.GridStat (run_util.py:153) ERROR: GridStat had 1 error.
01/30 06:53:25.810Z metplus INFO: Overwrite final conf here: /home/raj/software/met/met_plus/output/video/metplus_final.conf.20230130065325
01/30 06:53:25.811Z metplus (run_util.py:203) ERROR: METplus has finished running as user raj(743) but had 2 errors.
01/30 06:53:25.811Z metplus INFO: Check the log file for more information: /home/raj/software/met/met_plus/output/video/logs/metplus.log.20230130065325

[config]
PROCESS_LIST = GridStat
OMP_NUM_THREADS = 1
LOOP_BY = INIT
INIT_TIME_FMT = %Y%m%d%H
INIT_BEG = 2005080700
INIT_END = 2005080700
INIT_INCREMENT = 12H
LEAD_SEQ = 12
FCST_GRID_STAT_INPUT_DIR = {INPUT_BASE}/met_test/data/sample_fcst
FCST_GRID_STAT_INPUT_TEMPLATE = {init?fmt=%Y%m%d%H}/wrfprs_ruc13_{lead?fmt=%HH}.tm00_G212
OBS_GRID_STAT_INPUT_DIR = {INPUT_BASE}/met_test/new
OBS_GRID_STAT_INPUT_TEMPLATE = ST2ml{valid?fmt=%Y%m%d%H}_A03h.nc
GRID_STAT_CLIMO_MEAN_INPUT_DIR =
GRID_STAT_CLIMO_MEAN_INPUT_TEMPLATE =
GRID_STAT_CLIMO_STDEV_INPUT_DIR =
GRID_STAT_CLIMO_STDEV_INPUT_TEMPLATE =
GRID_STAT_OUTPUT_DIR = {OUTPUT_BASE}/met_tool_wrapper/GridStat/GridStat
GRID_STAT_OUTPUT_TEMPLATE = {init?fmt=%Y%m%d%H}
MODEL = WRF
OBTYPE = MC_PCP
GRID_STAT_ONCE_PER_FIELD = False
FCST_IS_PROB = false
FCST_VAR1_NAME = APCP
FCST_VAR1_LEVELS = A03
FCST_VAR1_THRESH = gt12.7, gt25.4, gt50.8, gt76.2
OBS_VAR1_NAME = APCP_03
OBS_VAR1_LEVELS = "(*,*)"
OBS_VAR1_THRESH = gt12.7, gt25.4, gt50.8, gt76.2
GRID_STAT_CONFIG_FILE = {PARM_BASE}/met_config/GridStatConfig_wrapped
GRID_STAT_REGRID_TO_GRID = NONE
GRID_STAT_DESC = NA
FCST_GRID_STAT_FILE_WINDOW_BEGIN = 0
FCST_GRID_STAT_FILE_WINDOW_END = 0
OBS_GRID_STAT_FILE_WINDOW_BEGIN = 0
OBS_GRID_STAT_FILE_WINDOW_END = 0
GRID_STAT_NEIGHBORHOOD_WIDTH = 1
GRID_STAT_NEIGHBORHOOD_SHAPE = SQUARE
GRID_STAT_NEIGHBORHOOD_COV_THRESH = >=0.5
GRID_STAT_OUTPUT_PREFIX = {MODEL}_{CURRENT_FCST_NAME}_vs_{OBTYPE}_{CURRENT_OBS_NAME}
GRID_STAT_OUTPUT_FLAG_CTC = STAT
GRID_STAT_OUTPUT_FLAG_CTS = STAT
GRID_STAT_OUTPUT_FLAG_ECLV = BOTH
GRID_STAT_OUTPUT_FLAG_GRAD = BOTH
GRID_STAT_NC_PAIRS_FLAG_LATLON = FALSE
GRID_STAT_NC_PAIRS_FLAG_RAW = FALSE
GRID_STAT_NC_PAIRS_FLAG_DIFF = FALSE
GRID_STAT_NC_PAIRS_FLAG_CLIMO = FALSE
GRID_STAT_NC_PAIRS_FLAG_APPLY_MASK = FALSE
GRID_STAT_SKIP_TIMES =
SKIP_TIMES =
GRID_STAT_MANDATORY = True
GRID_STAT_SKIP_IF_OUTPUT_EXISTS = False
OBS_GRID_STAT_INPUT_DATATYPE =
FCST_GRID_STAT_INPUT_DATATYPE =
GRID_STAT_CLIMO_MEAN_USE_FCST = False
GRID_STAT_CLIMO_MEAN_USE_OBS = False
GRID_STAT_CLIMO_STDEV_USE_FCST = False
GRID_STAT_CLIMO_STDEV_USE_OBS = False
FCST_GRID_STAT_PROB_THRESH = ==0.1
OBS_GRID_STAT_PROB_THRESH = ==0.1
INIT_SEQ =

[user_env_vars]

[runtime]
RUN_ID = c6e4d486
CLOCK_TIME = 20230130122325
METPLUS_VERSION = 5.0.0
MET_INSTALL_DIR = /home/raj/software/met/met_python/tar_files
CONFIG_INPUT = /home/raj/software/met/met_plus/METplus-5.0.0/parm/metplus_config/defaults.conf,/home/raj/software/met/met_plus/METplus-5.0.0/parm/use_cases/met_tool_wrapper/GridStat/GridStat.conf,/home/raj/software/met/met_plus/elogin.raj.video.conf
METPLUS_CONF = {OUTPUT_BASE}/metplus_final.conf.{LOG_TIMESTAMP}
TMP_DIR = {OUTPUT_BASE}/tmp
STAGING_DIR = {OUTPUT_BASE}/stage
FILE_LISTS_DIR = {STAGING_DIR}/file_lists.{LOG_TIMESTAMP}
CONVERT = convert
GEMPAKTOCF_JAR = GempakToCF.jar
GFDL_TRACKER_EXEC = /path/to/standalone_gfdl-vortextracker_v3.9a/trk_exec
INPUT_MUST_EXIST = True
USER_SHELL = bash
DO_NOT_RUN_EXE = False
SCRUB_STAGING_DIR = True
MET_BIN_DIR = {MET_INSTALL_DIR}/bin
INPUT_BASE = /home/raj/software/met/met_plus/METplus-3.1_sample_data/
OUTPUT_BASE = /home/raj/software/met/met_plus/output/video
LOG_METPLUS = /home/raj/software/met/met_plus/output/video/logs/metplus.log.20230130065325
LOG_DIR = {OUTPUT_BASE}/logs
LOG_TIMESTAMP_TEMPLATE = %Y%m%d%H%M%S
LOG_TIMESTAMP_USE_DATATIME = no
LOG_MET_OUTPUT_TO_METPLUS = yes
LOG_LEVEL = INFO
LOG_MET_VERBOSITY = 2
LOG_LINE_FORMAT = %(asctime)s.%(msecs)03dZ %(name)s (%(filename)s:%(lineno)d) %(levelname)s: %(message)s
LOG_ERR_LINE_FORMAT = {LOG_LINE_FORMAT}
LOG_DEBUG_LINE_FORMAT = {LOG_LINE_FORMAT}
LOG_INFO_LINE_FORMAT = %(asctime)s.%(msecs)03dZ %(name)s %(levelname)s: %(message)s
LOG_LINE_DATE_FORMAT = %m/%d %H:%M:%S
LOG_TIMESTAMP = 20230130065325
METPLUS_BASE = /home/raj/software/met/met_plus/METplus-5.0.0
PARM_BASE = /home/raj/software/met/met_plus/METplus-5.0.0/parm
LOG_GRID_STAT_VERBOSITY = 2

**Reply by jprestop** [reply]
Unfortunately, there is still no file attached. I am wondering if you are replying via email? If so, instead of replying via email, please go directly to the GitHub Discussion (https://github.com/dtcenter/METplus/discussions/2012) and attach it there by dragging and dropping your file into the area where you type.

**Reply by raj879423** [reply]
Hi @jprestop.

Yes, You are correct. I think you are unable to see my attached log because i was drooping email.
As per your suggestion I have attached through GitHub.

I am so sorry for big mistake.

[cat elogin.raj.video.conf.txt](https://github.com/dtcenter/METplus/files/10553553/cat.elogin.raj.video.conf.txt)
[metplus.log.20230130065325.txt](https://github.com/dtcenter/METplus/files/10553554/metplus.log.20230130065325.txt)
[metplus_final.conf.20230130065325.txt](https://github.com/dtcenter/METplus/files/10553555/metplus_final.conf.20230130065325.txt)

![METplus_dir_details](https://user-images.githubusercontent.com/67996756/215958227-558163ce-1f15-4774-a1bd-4841bfd80cdb.PNG)
![met_error](https://user-images.githubusercontent.com/67996756/215958270-56acd64d-c628-485d-91ce-053ba60cea0e.PNG)

**Reply by raj879423** [reply]
Hello @jprestop,

I really thanks to you specially for your support and finding my missing directory from above error.
I had checked as per your mentioned path and found missing directory call "met_test", so I created the directory and put "data" directory inside "met_test" and it got compiled successfully.

As per my understanding below are my compiled script run successfully, Please check and suggest me whether i have done successfully or not.

$ /home/raj/software/met/met_plus/METplus-5.0.0/ush/run_metplus.py -c /home/raj/software/met/met_plus/METplus-5.0.0/parm/use_cases/met_tool_wrapper/GridStat/GridStat.conf -c /home/raj/software/met/met_plus/elogin.raj.video.conf
Running METplus 5.0.0
Starting METplus v5.0.0
02/02 12:10:32.125 metplus (config_metplus.py:102) INFO: Starting METplus configuration setup.
02/02 12:10:32.147 metplus (config_metplus.py:230) INFO: Parsing config file: /home/raj/software/met/met_plus/METplus-5.0.0/parm/metplus_config/defaults.conf
02/02 12:10:32.160 metplus (config_metplus.py:230) INFO: Parsing config file: /home/raj/software/met/met_plus/METplus-5.0.0/parm/use_cases/met_tool_wrapper/GridStat/GridStat.conf
02/02 12:10:32.170 metplus (config_metplus.py:230) INFO: Parsing config file: /home/raj/software/met/met_plus/elogin.raj.video.conf
02/02 12:10:32.190 metplus (config_metplus.py:304) INFO: Adding LOG_TIMESTAMP='20230202121032'
02/02 12:10:32.191 metplus (config_metplus.py:335) INFO: Replace LOG_METPLUS with '/home/raj/software/met/met_plus/output/video/logs/metplus.log.20230202121032'
02/02 12:10:32.194Z metplus INFO: Running METplus v5.0.0 as user raj(743) with command: /home/raj/software/met/met_plus/METplus-5.0.0/ush/run_metplus.py -c /home/raj/software/met/met_plus/METplus-5.0.0/parm/use_cases/met_tool_wrapper/GridStat/GridStat.conf -c /home/raj/software/met/met_plus/elogin.raj.video.conf
02/02 12:10:32.194Z metplus INFO: Log file: /home/raj/software/met/met_plus/output/video/logs/metplus.log.20230202121032
02/02 12:10:32.194Z metplus INFO: METplus Base: /home/raj/software/met/met_plus/METplus-5.0.0
02/02 12:10:32.194Z metplus INFO: Final Conf: /home/raj/software/met/met_plus/output/video/metplus_final.conf.20230202121032
02/02 12:10:32.194Z metplus INFO: Config Input: /home/raj/software/met/met_plus/METplus-5.0.0/parm/metplus_config/defaults.conf
02/02 12:10:32.194Z metplus INFO: Config Input: /home/raj/software/met/met_plus/METplus-5.0.0/parm/use_cases/met_tool_wrapper/GridStat/GridStat.conf
02/02 12:10:32.194Z metplus INFO: Config Input: /home/raj/software/met/met_plus/elogin.raj.video.conf
02/02 12:10:32.227Z metplus INFO: ****************************************
02/02 12:10:32.227Z metplus INFO: * Running METplus GridStatWrapper
02/02 12:10:32.227Z metplus INFO: *  at init time: 2005-08-07 00:00
02/02 12:10:32.227Z metplus INFO: ****************************************
02/02 12:10:32.227Z metplus INFO: Processing forecast lead 12 hours
02/02 12:10:32.233Z metplus INFO: ENVIRONMENT FOR NEXT COMMAND:
02/02 12:10:32.233Z metplus INFO: CLIMO_MEAN_FILE=
02/02 12:10:32.233Z metplus INFO: CLIMO_STDEV_FILE=
02/02 12:10:32.233Z metplus INFO: FCST_FIELD={ name="APCP"; level="A03"; cat_thresh=[ gt12.7,gt25.4,gt50.8,gt76.2 ]; }
02/02 12:10:32.233Z metplus INFO: FCST_TIME=012
02/02 12:10:32.233Z metplus INFO: INPUT_BASE=/home/raj/software/met/met_plus/METplus-3.1_sample_data/
02/02 12:10:32.233Z metplus INFO: METPLUS_CENSOR_THRESH=
02/02 12:10:32.234Z metplus INFO: METPLUS_CENSOR_VAL=
02/02 12:10:32.234Z metplus INFO: METPLUS_CLIMO_CDF_DICT=
02/02 12:10:32.234Z metplus INFO: METPLUS_CLIMO_MEAN_DICT=
02/02 12:10:32.234Z metplus INFO: METPLUS_CLIMO_STDEV_DICT=
02/02 12:10:32.234Z metplus INFO: METPLUS_DESC=desc = "NA";
02/02 12:10:32.234Z metplus INFO: METPLUS_DISTANCE_MAP_DICT=
02/02 12:10:32.234Z metplus INFO: METPLUS_FCST_FIELD=field = [{ name="APCP"; level="A03"; cat_thresh=[ gt12.7,gt25.4,gt50.8,gt76.2 ]; }];
02/02 12:10:32.234Z metplus INFO: METPLUS_FCST_FILE_TYPE=
02/02 12:10:32.234Z metplus INFO: METPLUS_FOURIER_DICT=
02/02 12:10:32.234Z metplus INFO: METPLUS_GRID_WEIGHT_FLAG=
02/02 12:10:32.234Z metplus INFO: METPLUS_HSS_EC_VALUE=
02/02 12:10:32.234Z metplus INFO: METPLUS_INTERP_DICT=
02/02 12:10:32.234Z metplus INFO: METPLUS_MASK_DICT=
02/02 12:10:32.234Z metplus INFO: METPLUS_MET_CONFIG_OVERRIDES=
02/02 12:10:32.234Z metplus INFO: METPLUS_MODEL=model = "WRF";
02/02 12:10:32.234Z metplus INFO: METPLUS_NBRHD_COV_THRESH=cov_thresh = [>=0.5];
02/02 12:10:32.234Z metplus INFO: METPLUS_NBRHD_SHAPE=shape = SQUARE;
02/02 12:10:32.235Z metplus INFO: METPLUS_NBRHD_WIDTH=width = [1];
02/02 12:10:32.235Z metplus INFO: METPLUS_NC_PAIRS_FLAG_DICT=nc_pairs_flag = {latlon = FALSE;raw = FALSE;diff = FALSE;climo = FALSE;apply_mask = FALSE;}
02/02 12:10:32.235Z metplus INFO: METPLUS_NC_PAIRS_VAR_NAME=
02/02 12:10:32.235Z metplus INFO: METPLUS_OBS_FIELD=field = [{ name="APCP_03"; level="(*,*)"; cat_thresh=[ gt12.7,gt25.4,gt50.8,gt76.2 ]; }];
02/02 12:10:32.235Z metplus INFO: METPLUS_OBS_FILE_TYPE=
02/02 12:10:32.235Z metplus INFO: METPLUS_OBTYPE=obtype = "MC_PCP";
02/02 12:10:32.235Z metplus INFO: METPLUS_OUTPUT_FLAG_DICT=output_flag = {ctc = STAT;cts = STAT;eclv = BOTH;grad = BOTH;}
02/02 12:10:32.235Z metplus INFO: METPLUS_OUTPUT_PREFIX=output_prefix = "WRF_APCP_vs_MC_PCP_APCP_03";
02/02 12:10:32.235Z metplus INFO: METPLUS_REGRID_DICT=regrid = {to_grid = NONE;}
02/02 12:10:32.235Z metplus INFO: METPLUS_SEEPS_P1_THRESH=
02/02 12:10:32.235Z metplus INFO: MET_TMP_DIR=/home/raj/software/met/met_plus/output/video/tmp
02/02 12:10:32.235Z metplus INFO: MODEL=WRF
02/02 12:10:32.235Z metplus INFO: NEIGHBORHOOD_COV_THRESH=[>=0.5]
02/02 12:10:32.235Z metplus INFO: NEIGHBORHOOD_SHAPE=SQUARE
02/02 12:10:32.235Z metplus INFO: NEIGHBORHOOD_WIDTH=1
02/02 12:10:32.235Z metplus INFO: OBS_FIELD={ name="APCP_03"; level="(*,*)"; cat_thresh=[ gt12.7,gt25.4,gt50.8,gt76.2 ]; }
02/02 12:10:32.235Z metplus INFO: OBTYPE=MC_PCP
02/02 12:10:32.236Z metplus INFO: OMP_NUM_THREADS=1
02/02 12:10:32.236Z metplus INFO: OUTPUT_PREFIX=WRF_APCP_vs_MC_PCP_APCP_03
02/02 12:10:32.236Z metplus INFO: REGRID_TO_GRID=NONE
02/02 12:10:32.236Z metplus INFO: VERIF_MASK=[]
02/02 12:10:32.236Z metplus INFO: COMMAND: /home/raj/software/met/met_python/tar_files/bin/grid_stat -v 2 /home/raj/software/met/met_plus/METplus-3.1_sample_data/met_test/data/sample_fcst/2005080700/wrfprs_ruc13_12.tm00_G212 /home/raj/software/met/met_plus/METplus-3.1_sample_data/met_test/new/ST2ml2005080712_A03h.nc /home/raj/software/met/met_plus/METplus-5.0.0/parm/met_config/GridStatConfig_wrapped -outdir /home/raj/software/met/met_plus/output/video/met_tool_wrapper/GridStat/GridStat/2005080700
02/02 12:10:33.116Z metplus INFO: Overwrite final conf here: /home/raj/software/met/met_plus/output/video/metplus_final.conf.20230202121032
02/02 12:10:33.117Z metplus INFO: Check the log file for more information: /home/raj/software/met/met_plus/output/video/logs/metplus.log.20230202121032
02/02 12:10:33.117Z metplus INFO: METplus has successfully finished running as user raj(743).
raj@elogin04:~/software/met/met_plus> echo $?

=======================================================================

[metplus.log.20230202121032.txt](https://github.com/dtcenter/METplus/files/10577508/metplus.log.20230202121032.txt)

 

Thankyou so much for your kind help and support!

**Reply by jprestop** [reply]
Great news, @raj879423!  As you indicated, it looks like your script completed successfully.  

Please feel free to select "Mark as answer" for whichever answer you feel is best, if you have no further questions.  After which, I will lock the Discussion to prevent future posts in this Discussion.  Feel free to start a new Discussion with any other questions or issues you encounter.

**Comment by jprestop** [comment, marked as answer]
Hi @raj879423.  Thank you!  I am now able to see the files.

Could you please run the following to see if this file exists and send the output of this command?
```
ls /home/raj/software/met/met_plus/METplus-3.1_sample_data/met_test/data/sample_fcst/2005080700/wrfprs_ruc13_12.tm00_G212
```

**Reply by raj879423** [reply]
Hello @jprestop,

Yes file is exists.

$ ls /home/raj/software/met/met_plus/METplus-3.1_sample_data/met_test/data/sample_fcst/2005080700/
wrfprs_ruc13_00.tm00_G212  wrfprs_ruc13_06.tm00_G212  wrfprs_ruc13_12.tm00_G212  wrfprs_ruc13_18.tm00_G212  wrfprs_ruc13_24.tm00_G212
wrfprs_ruc13_03.tm00_G212  wrfprs_ruc13_09.tm00_G212  wrfprs_ruc13_15.tm00_G212  wrfprs_ruc13_21.tm00_G212


You can close this discussion as of now.
 
Thanks you!
